Gems/Multiplayer/Code/Source/NetworkEntity/EntityReplication/EntityReplicator.cpp

```diff
@@ -321,6 +326,11 @@ namespace Multiplayer
 
     void EntityReplicator::MarkForRemoval()
     {
+        // NOTE: It's possible that heading into this function, m_entityHandle is already invalid if this was triggered
+        // via OnEntityRemoved.
+
+        AZLOG(NET_RepDeletes, "Marking entity %llu for removal.", (AZ::u64)m_entityHandle.GetNetEntityId());
```

Comment:
Prefer C++ casts?  The AZLOGs are using c-style casts, but the other logs are using static_cast<>